### DISCUSSION TITLE
Is the Discussion tab here the best electronci paper trail for XLIFF development?

### DISCUSSION THREAD

**Discussion by bryanschnabel** [oasis-tcs/xliff-xliff-22#8] [Q&A]
I'm not sure if I agree. What are your thoughts?

Bryan

**Comment by rmraya** [comment]
I'd rather use the TC mailing list. Outlook is where I keep my archive.

Rodolfo

**Reply by bryanschnabel** [reply]
Thanks, Rodolfo. I am in agreement. I actually am not in favor of github for this purpose. The OASIS archives seem more official. Having a github account does not feel like an appropriate requirement for somebody trying to track XLIFF's progress.
